lib/tasks/specs.rake

```diff
@@ -0,0 +1,47 @@
+namespace :openfoodnetwork do
```

Comment:
@sauloperez we wanted to get rid of this namespace in the rake tasks right?
it was discussed in @mkllnk's sample_data PR #3209. we should keep both tasks coherent in terms of namespaces. do you guys know what's the best approach here?

Reply:
Thanks for pointing out. I thought about it but I prefered to not annoy again with that topic. My opinion is that we should get rid of it.

Reply:
lets get rid of it then :-D

Reply:
Let's have the discussion on https://community.openfoodnetwork.org/t/rake-task-naming/1531.